core/src/main/java/org/elasticsearch/indices/ttl/IndicesTTLService.java

```
@@ -340,7 +340,7 @@ public void await() {
             try {
                 condition.await(timeout.millis(), TimeUnit.MILLISECONDS);
             } catch (InterruptedException e) {
-                Thread.interrupted();
+                Thread.currentThread().interrupt();
```

Ok, I'll implement an empty catch block and just put a comment there.